tests/release/const.go

```diff
@@ -1,22 +1,23 @@
 package release
 
-// import (
-// 	"time"
-// )
+import (
+	"time"
+)
 
-// const (
-// 	snapshotName          string = "snapshot"
-// 	sourceReleaseLinkName string = "source-release-link"
-// 	targetReleaseLinkName string = "target-release-link"
-// 	releaseStrategyName   string = "strategy"
-// 	releaseName           string = "release"
-// 	releasePipelineName   string = "release-pipeline"
-// 	applicationName       string = "application"
-// 	releasePipelineBundle string = "quay.io/hacbs-release/demo:m5-alpine"
-// 	releaseStrategyPolicy string = "policy"
+const (
+	snapshotName                  = "snapshot"
+	sourceReleasePlanName         = "source-release-plan"
+	destinationReleasePAName      = "sre-production"
```

Comment:
maybe we can call this destinationRPAName if you want it short, or destinationReleasePlanAdmissionName if not. I had to scratch my head a bit to realize what a ReleasePAN was